backend/sharelinks/views.py

```
@@ -1781,7 +1782,7 @@ def post(self, request, slug, *args, **kwargs):
                 status=status.HTTP_500_INTERNAL_SERVER_ERROR,
             )
 
-        viewer, _ = Viewer.objects.get_or_create(
+        viewer, _created = Viewer.objects.get_or_create(
```

Ruff (0.16.1)

[warning] Unpacked variable `viewer` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)